bitnami/mastodon/values.yaml

```diff
@@ -170,14 +190,83 @@ enableSearches: true
 ##
 enableS3: true
 
-## @param webDomain Web domain for Mastodon
+## @param local_https Set this instance to advertise itself to the fediverse using HTTPS rather than HTTP URLs. This should almost always be true.
+##
+local_https: true
+
+## @param localDomain The domain name used by accounts on this instance. Unless you're using
+## webDomain, this value should be set to the URL at which your instance is hosted.
+##
+localDomain: ""
+
+## @param webDomain Optional alternate domain used when you want to host Mastodon at a
+## different URL than localDomain. This value should only be set if you need it, and
+## cannot be changed later. Consult the Mastodon documentation before using webDomain:
+## https://docs.joinmastodon.org/admin/config/#federation
 ##
 webDomain: ""
 
+## @param defaultLocale Set the default locale for this instance
+##
+defaultLocale: en
+
 ## @param s3AliasHost S3 alias host for Mastodon (will use http://webDomain/bucket if not set)
 ##
 s3AliasHost: ""
 
+## @section Mastodon SMTP Parameters
```

Comment:
If you check other charts, SMTP parameters are included in `common parameters` section

Reply:
Thanks, removed the SMTP section 👍